docs/platforms/react-native/sourcemaps/uploading/hermes.mdx

```diff
@@ -161,4 +161,10 @@ When uploaded, the `bundle` file will have a size of 0 bytes. This is expected w
 
 </Alert>
 
-For more Hermes guides, see the [Hermes Throubleshooting](/platforms/react-native/manual-setup/hermes/#throubleshooting) docs section.
+## Troubleshooting
+
+When generated source maps using this guide don’t seem to be correct, check the `Contexts` -> `React Native` -> `Hermes Debug Info` flag in the events. If the flag is `true`, this means the bundle shipped with the native application package was generated without Hermes source maps, and plain Metro (JS Core) source maps need to be used for the symbolication. Follow the [JavaScript Core guide](/platforms/react-native/sourcemaps/uploading/jsc/) to upload the correct source maps.
+
+Note that the debug information included in the Hermes bundle increases the size of the final shipped bundle, and it's recommended to enable the source maps during the native application build (even when uploaded later manually, as generating the Hermes source maps has a side effect of striping the debug information, saving it to the source map). For automatic source maps upload, follow the [Manual Setup guide](/platforms/react-native/manual-setup/manual-setup/). The Sentry Xcode Scripts and Gradle Plugin automatically enable source maps. If you can't use the automatic upload scripts, enable source maps as described in the [React Native Source Maps guide](https://reactnative.dev/docs/debugging-release-builds#enabling-source-maps).
+
+For more Hermes guides, see the [Hermes Troubleshooting](/platforms/react-native/manual-setup/hermes/#throubleshooting) docs section.
```

Comment:
The following link typo should fix the check failure 🤞 

```suggestion
For more Hermes guides, see the [Hermes Troubleshooting](/platforms/react-native/manual-setup/hermes/#troubleshooting) docs section.
```

Other than that LGTM 🙇

Reply:
Good catch, ty!